### ISSUE TITLE
Credit invoice with Klarna not posible

### ISSUE THREAD

**Issue by tropilex** [mollie/magento2#221]
You cannot make a credit invoice with Klarna.

When you ship the order with Magento the order gets the status 'Closed' back from Mollie.

We need to make credit invoices with Magento when we have a return. The (credit)invoices are synced with our administration (Exact Online). So now Exact still thinks the order is completed and we pay the taxes and all.

I understand it's difficult (for now?) to make a full credit with Mollie and get the whole administration right with Klarna. But we need to have the credit invoice button back.

![Screenshot-2020-03-04-at-16 16 21](https://user-images.githubusercontent.com/53861858/75895367-d818c100-5e35-11ea-8305-2ea9f1e100ba.png)
![Screenshot-2020-03-04-at-16 31 12](https://user-images.githubusercontent.com/53861858/75895371-d949ee00-5e35-11ea-9d22-dcc0f35dd265.png)

**Comment by Frank-Magmodules** [comment]
Hi @tropilex , Thank you for opening this issue. We can't directly reproduce this on our latest version using the following steps;

1. Simulate a Klarna Order and Authorize the Payment
2. Ship the order
3. Open the paid invoice and create a credit memo

After this, the order is closed and the credit-memo is successfully created. Can you please check if you are using the latest Mollie version? In case the issue still exists, please contact us through our [contact form](https://www.magmodules.eu/support-form.html) so we can exchange log files to check the issue?
Thank you!

**Comment by Frank-Magmodules** [comment]
Hi @tropilex, do you have any update on this issue?

**Comment by tropilex** [comment]
Yes @Frank-Magmodules . It 's still not working. I ask Mollie and they have the same problem. See attachment.

![Screenshot 2020-03-16 at 16 11 27](https://user-images.githubusercontent.com/53861858/76772333-fd47f080-67a0-11ea-80dd-523c2a49d8c3.png)

![Screenshot 2020-03-16 at 16 12 01](https://user-images.githubusercontent.com/53861858/76772326-fa4d0000-67a0-11ea-8dd6-9c8878119b6a.png)

**Comment by Frank-Magmodules** [comment]
HI @tropilex ,

Can you please share your log files on this through our [contact form](https://www.magmodules.eu/support-form.html/)?  Thank you!

**Comment by tropilex** [comment]
@Frank-Magmodules 

I went back into the problem and I've good news: It's working again. 

The problem arose on 26-01-2020. And from 25-02-2020 all Klarna orders got the correct status again from Mollie when it was shipped.

I checked our release logs and we didn't change anything around both dates. The weird thing is that the status really came from Mollie, so what exactly was going on....

But I'm glad it's working again!
